packages/components/input/__tests__/textarea.test.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import {render} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import {Textarea} from "../src";
+
+describe("Textarea", () => {
```

Comment:
add one more test case. define `onClear` without defining `isClearable`. It should have clear button as well.

Reply:
OK